### PR TITLE
docs(angular/lightbox): remove obsolete button alignment infos

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

**Review by kyubisation — CHANGES_REQUESTED** [review]
As far as I know, the specification of lightbox is stricter in regard to button placement.
I don't know whether we should support this?

File: src/angular/lightbox/lightbox-content-directives.ts [hunk 1]
```diff
@@ -132,6 +132,16 @@ export class SbbLightboxContent {}
...
+    '[class.sbb-lightbox-actions-align-center]': 'align === "center"',
+    '[class.sbb-lightbox-actions-align-end]': 'align === "end"',
+  },
 })
-export class SbbLightboxActions {}
+export class SbbLightboxActions {
+  /**
+   * Horizontal alignment of action buttons.
+   */
+  @Input() align?: 'start' | 'center' | 'end' | 'space between' = 'space between';
```
Comment (mhaertwig) [inline src/angular/lightbox/lightbox-content-directives.ts:146]: Do we need the question mark after `align` if there's a default value?
I would prefer `space-between` (with `-`).

**Comment by jeripeierSBB** [comment]
@mhaertwig We decided to not introduce this feature, but make a doc fix instead.

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Comment by mhaertwig** [comment]
LGTM 👍

**Pull request merged**